static/css/main.css

```diff
@@ -83,6 +83,7 @@ blockquote {
 
 pre {
   padding-left: 1.5rem;
+  overflow-x: auto;
```

Comment:
```suggestion
  white-space: pre-wrap;
  word-wrap: break-word;
```

The `overflow-x: auto;` property adds a horizontal scrollbar to the block if the content is wider than the block. However, it only works for block elements, whereas by default `<pre>` is an inline element.

Also, perhaps the scroll bar is not the most convenient solution. Part of the code will be invisible, and the user will have to do additional manipulations to see the whole code:

![image](https://user-images.githubusercontent.com/3881568/66004379-f6eee200-e4a8-11e9-9d28-bff8c5c7b0ce.png)

Instead, I suggest using the properties `white-space: pre-wrap;` and `word-wrap: break-word;`. The first property wraps words on the next line, and the second property splits too long words:

![image](https://user-images.githubusercontent.com/3881568/66004353-e76f9900-e4a8-11e9-82d8-7b6c26382e77.png)

Reply:
Hey @glebkema thanks for the review and comments. However current browsers do display `<pre>` element as a block. Which allows the overflow to work. 
![pre styles](https://user-images.githubusercontent.com/904501/66022161-a1095100-e51f-11e9-87f5-e0e9e074d8c1.png)

I do like your approach with the word-wrap but I think it's more useful on specific situations where contents of `<pre>` will still make sense when styled that way but shouldn't be applied globally to prevent unexpected behavior. i.e. this handles the markdown really well but won't for all content of `<pre>`. Here's a [sample code](https://jsbin.com/relamujuyi/edit?html,css,output) i've included with both styles, you'll see how illegible some of the content is with the suggested styles.

PS: Scrollbars (even horizontal ones) are part of the web UI and are useful in some cases, IMO they are not additional manipulations for users. That's like saying all web pages should be full page/viewport so the user doesn't have to scroll.

Reply:
Hi @talentedunicorn Thanks for the detailed answer!

I agree that relatively large code elements are more readable with a scrollbar and without additional hyphenation. I proposed my idea because I saw only small <pre> blocks on the problem page.

And I can’t understand why yesterday my Chrome showed `<pre>` blocks as `inline`. Today I see the `display: block;` property of the user agent for the `<pre>` elements. Therefore currently I have no idea why your solution did not work.

Reply:
Thanks @glebkema I did like your approach but after some testing I realized it doesn't work for all scenarios. I am not sure why your Chrome didn't show the styles that way, are you using the latest version?
Thanks for reviewing, do you mind updating your review if this is clear now?

Reply:
Yes, I withdraw my suggestion and accept yours. 

As I see, you have already marked my review as resolved. Should I do something else with my review? Sorry, I'm not very good at how the GitHub works.